nessie/src/main/java/org/apache/iceberg/nessie/NessieTableOperations.java

```diff
@@ -66,6 +66,11 @@ public NessieTableOperations(
     this.fileIO = fileIO;
   }
 
+  @Override
+  protected String tableName() {
+    return key.toString();
```

Comment:
I took a look and this _seems_ to be derived from the table identifier (via joining the catalog / db name / table parts on `.`).

However, I've not used Nessie as a catalog so please let me know if this is not correct. I will need to test out Nessie some time for my own understanding 🙂